commands.js

```diff
@@ -351,6 +352,7 @@ var commands = exports.commands = {
 		}
 	},
 
+        alias: 'roomalias',
```

Comment:
No, it's too ambiguous and this shouldn't be used commonly enough to have this as an alias.